docs/excel/custom-functions-naming.md

```diff
@@ -32,6 +32,31 @@ To fix a name conflict, change the name in your add-in and try the function agai
 
 Also consider how you'd like people to use the functions within your add-in. In many cases, it makes sense to add multiple arguments to a function rather than create multiple functions with the same or similar names.
 
+## Localizing function names
```

Comment:
I think this duplicates most of the info already in the other article. Maybe instead is there anything to point out about naming and localization? For example, if you localize to Russian, do you need to avoid the Russian version of built-in commands? For the other info just link to the localization article.

Reply:
Will do!